MODULE_NOT_FOUND /node_modules/node/bin/node

Hi,

Once again, thank you for this tool.

Installing the `node-api-analytics` package throws the following error (node v18, v20 and v22):

```
➜ npm i -S node-api-analytics
npm ERR! code 1
npm ERR! path /Users/me/tests/fastify-analytics/node_modules/node
npm ERR! command failed
npm ERR! command sh -c node installArchSpecificPackage
npm ERR! npm ERR! code ETARGET
npm ERR! npm ERR! notarget No matching version found for node-bin-darwin-arm64@14.21.3.
npm ERR! npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! npm ERR! notarget a package version that doesn't exist.
npm ERR! 
npm ERR! npm ERR! A complete log of this run can be found in: /Users/me/.npm/_logs/2024-05-31T11_31_55_761Z-debug-0.log
npm ERR! node:internal/modules/cjs/loader:1148
npm ERR!   throw err;
npm ERR!   ^
npm ERR! 
npm ERR! Error: Cannot find module 'node-bin-darwin-arm64/package.json'
npm ERR! Require stack:
npm ERR! - /Users/me/tests/fastify-analytics/node_modules/node/installArchSpecificPackage.js
npm ERR!     at Module._resolveFilename (node:internal/modules/cjs/loader:1145:15)
npm ERR!     at Function.resolve (node:internal/modules/helpers:190:19)
npm ERR!     at ChildProcess.<anonymous> (/Users/me/tests/fastify-analytics/node_modules/node-bin-setup/index.js:19:27)
npm ERR!     at ChildProcess.emit (node:events:519:28)
npm ERR!     at maybeClose (node:internal/child_process:1105:16)
npm ERR!     at ChildProcess._handle.onexit (node:internal/child_process:305:5) {
npm ERR!   code: 'MODULE_NOT_FOUND',
npm ERR!   requireStack: [
npm ERR!     '/Users/me/tests/fastify-analytics/node_modules/node/installArchSpecificPackage.js'
npm ERR!   ]
npm ERR! }
npm ERR! 
npm ERR! Node.js v20.13.1

npm ERR! A complete log of this run can be found in: /Users/me/.npm/_logs/2024-05-31T11_31_52_852Z-debug-0.log
```

Repro steps:

1. Start a new project `npm init -y`
2. Install `npm i -S node-api-analytics` 

Thanks

--- a/analytics/nodejs/package.json
+++ b/analytics/nodejs/package.json
@@ -8,7 +8,6 @@
     "test": "tests"
   },
   "dependencies": {
-    "node": "^14.0.0",
     "node-fetch": "^3.0.0"
   },
   "scripts": {

